clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -1519,9 +1519,11 @@ static void InitializePredefinedMacros(const TargetInfo &TI,
   if (TI.getTriple().isOSBinFormatELF())
     Builder.defineMacro("__ELF__");
 
-  if (LangOpts.Sanitize.has(SanitizerKind::Address))
+  if (LangOpts.Sanitize.has(SanitizerKind::Address) ||
+      LangOpts.Sanitize.has(SanitizerKind::KernelAddress))
```

Comment:
This seems to be normally written `hasOneOf(SanitizerKind::Address | SanitizerKind::KernelAddress)`, similarly below.